lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -108,7 +108,7 @@ class ConstantStringMigrator extends GeneralizingAstVisitor
     if (node.isConst &&
         node.initializer != null &&
         node.initializer is SimpleStringLiteral) {
-      SimpleStringLiteral literal = node.initializer as SimpleStringLiteral;
+      SimpleStringLiteral literal = node.initializer! as SimpleStringLiteral;
```

Comment:
There were lints causing CI failures in a few misc. files that are unrelated to the scope of this PR.